.github/workflows/ci-scalardb-cluster-monitoring.yml

```diff
@@ -0,0 +1,194 @@
+name: CI for the ScalarDB Cluster Monitoring chart
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "charts/scalardb-cluster-monitoring/**"
+  push:
+    branches:
+      - main
+    paths:
+      - "charts/scalardb-cluster-monitoring/**"
+  workflow_dispatch:
+
+jobs:
+  lint-chart:
+    runs-on: ubuntu-latest
+    env:
+      PYTHON_VERSION: 3.13
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
Do we need to fetch all history? Fetching a single commit will speed up the job.

Reply:
Ah, I see. I just copied the existing YAML file, but I think we don't need to fetch all history in this new test. I will remove it. Thank you for pointing it out!